modules/site-health/admin-site-health.php

```diff
@@ -248,9 +260,45 @@ public function status_tests( $tests ) {
 				'test'  => array( $this, 'homepage_test' ),
 			);
 		}
+		$tests['direct']['pll_simplexml'] = array(
+			'label' => esc_html__( 'SimpleXML', 'polylang' ),
+			'test'  => array( $this, 'simplexml_available' ),
+		);
 		return $tests;
 	}
 
+	/**
+	 * Test if the SimpleXML PHP extension is available.
+	 *
+	 * @since 3.2
+	 *
+	 * @return array $result with test results.
```

Comment:
Looks like I forgot a word.

```suggestion
	 * @return array $result Array with test results.
```